src/main/java/seedu/address/model/event/EventContainsDatePredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.event;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+public class EventContainsDatePredicate implements Predicate<Event> {
+    private final List<String> dates;
+
+    /**
+     * Represents the Predicate for dates.
+     * @param dates date to search for.
+     */
+    public EventContainsDatePredicate(List<String> dates) {
+        this.dates = dates;
+        System.out.println(dates);
+    }
```

Comment:
Is `System.out.println(dates);` still being used for debugging? If not maybe we can remove it 😅

Reply:
Yup fixed